ledfx/effects/concentric.py

```
@@ -0,0 +1,144 @@
+            vol.Optional(
+                "idle_speed",
+                description="Speed of the effect when nothing happens.",
+                default=1,
+            ): vol.All(vol.Coerce(float), vol.Range(min=0.0, max=3.0)),
+        }
+    )
+
+    def on_activate(self, pixel_count):
+        super().on_activate(pixel_count)
+
+    def config_updated(self, config):
```

Ruff (0.12.2)

Unused method argument: `config`

(ARG002)